locales/locales.go

```diff
@@ -0,0 +1,30 @@
+package locales
+
+import (
+	"github.com/nicksnyder/go-i18n/v2/i18n"
+	"golang.org/x/text/language"
+	"gopkg.in/yaml.v2"
+)
+
+var supportedLanguages = []string{"en", "pt"}
```

Comment:
I think that can be moved to  `func init()`

Reply:
Do you think that would be better? I thought of this as a way of having supported languages be shown in a straightforward way, so that you don't really need to get too deep into code logic to support a new language.
Does that make sense to you?

Reply:
you're only using it in one place, it doesn't make sense for it to stay on top because it's not being exported

Reply:
alright, I've moved the `supportedLanguages` var to the init function;)